integration/integration_test.go

```diff
@@ -61,42 +60,13 @@ func Test(t *testing.T) {
 	check.Suite(&KubeDNSSuite{})
 	check.Suite(&HelmSuite{})
 
-	images = append(images, image{"containous/maesh:latest", false})
-	images = append(images, image{"containous/whoami:v1.0.1", true})
-	images = append(images, image{"containous/whoamitcp:v0.0.2", true})
-	images = append(images, image{"containous/whoamiudp:v0.0.1", true})
-	images = append(images, image{"coredns/coredns:1.2.6", true})
-	images = append(images, image{"coredns/coredns:1.3.1", true})
-	images = append(images, image{"coredns/coredns:1.4.0", true})
-	images = append(images, image{"coredns/coredns:1.5.2", true})
-	images = append(images, image{"coredns/coredns:1.6.3", true})
-	images = append(images, image{"coredns/coredns:1.7.0", true})
-	images = append(images, image{"giantswarm/tiny-tools:3.9", true})
-	images = append(images, image{"gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.7", true})
-	images = append(images, image{"gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.7", true})
-	images = append(images, image{"gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.7", true})
-	images = append(images, image{"traefik:v2.3", true})
-
-	for _, image := range images {
-		if image.pull {
-			cmd := exec.Command("docker", "pull", image.name)
-			cmd.Env = os.Environ()
-
-			output, err := cmd.CombinedOutput()
-			fmt.Println(string(output))
-
-			if err != nil {
-				fmt.Printf("unable to pull docker image: %v", err)
-			}
-		}
-	}
-
 	check.TestingT(t)
 }
 
 type image struct {
-	name string
-	pull bool
+	repository string
```

Comment:
Here I would have kept the property `name` as it is usally the way to reference a docker image.
wdyt?